serde/src/de/impls.rs

```diff
@@ -1825,3 +1825,20 @@ where
         deserializer.deserialize_enum("Result", VARIANTS, ResultVisitor(PhantomData))
     }
 }
+
+////////////////////////////////////////////////////////////////////////////////
+
+#[cfg(feature = "std")]
+use std::num::Wrapping;
+
+#[cfg(feature = "std")]
+impl<'de, T> Deserialize<'de> for Wrapping<T>
+where 
+    T: Deserialize<'de> 
+{
+    fn deserialize<D>(deserializer: D) -> Result<Wrapping<T>, D::Error> 
+        where D: Deserializer<'de>
+    {
+        Deserialize::deserialize(deserializer).map(Wrapping)
```

Comment:
Deserializing a `Wapping<u8>` from `500` will fail. Does that match your use case?

Reply:
Can you clarify? Do you mean that if you deserialize a number that exceeds the max for that type it won't wrap? That's fine, I didn't expect it to use the wrapping functionality at deserialization time.

Reply:
Then isn't this something `Wrapping(u8::deserialize(deserializer)?)` can already accomplish without this impl?

Reply:
Yea that's what I meant. Just wanted to make sure. It's backwards compatible to change later if we want to.

Reply:
@hcpl If you just want to (de)serialize a `Wrapping` that's fine, but it becomes somewhat more problematic if you want to `Derive` a struct containing one. I guess you could decorate every field with a wrapping with a `with` attribute, but it's overall cleaner to do this.

Reply:
So there are 2 desired behaviours for `Wrapping`: accept and produce values in the certain range, or only do so for producing values while accepting anything.

Logically speaking, if we were to provide wrapping behaviour in `impl Deserialize`, we would have to do custom number parsing to wrap _any_ possible number for consistency's sake.

Also, `Wrapping<T>` doesn't have any bounds on `T`, so we end up having to rely on the unstable specialization feature for numeric types.

So yeah, easier to leave things this way.